www/templates/foundation/members/index.html

```diff
@@ -50,9 +50,17 @@ <h1 class="display-5 fw-bold mb-4">
                         <p>
                             {% blocktranslate trimmed %}
                             The merit of each application will be reviewed on a case-by-case basis by the Board or by a
-                            committee appointed for this purpose, according to the Bylaws established by the Board.
+                            committee appointed for this purpose, according to the 
+                            <a href="https://almalinux.org/p/foundation-bylaws/" id="al-laws-link">
```

Comment:
- Links should not be part of blocktranslate. Please see other templates on examples how to handle links in translatable text.
- Styling for inline links should be anchored from parent element (e.g. styling the `p`) not using an inline ID.